tests/integration/run_tests.py

```diff
@@ -442,6 +435,64 @@ async def test_crud_users_works():
             assert resp.status == 404, "HTTP Status code error"
 
 
+async def test_submissions_work():
+    """Test actions in submission xml files."""
+    async with aiohttp.ClientSession() as sess:
+        # Post original submission with two 'add' actions
+        sub_files = [("submission", "ERA521986_valid.xml"), ("study", "SRP000539.xml"), ("sample", "SRS001433.xml")]
+        data = await create_multi_file_request_data(sub_files)
+        async with sess.post(f"{submit_url}", data=data) as resp:
+            LOG.debug("Checking initial submission worked")
+            assert resp.status == 200, "HTTP Status code error"
+            res = await resp.json()
+            assert len(res) == 2, "content mismatch"
+            assert res[0]["schema"] == "study", "content mismatch"
+            assert res[1]["schema"] == "sample", "content mismatch"
+            study_access_id = res[0]["accessionId"]
+
+        # Sanity check that the study object was inserted correctly before modifying it
+        async with sess.get(f"{base_url}/study/{study_access_id}") as resp:
+            LOG.debug("Sanity checking that previous object was added correctly")
+            assert resp.status == 200, "HTTP Status code error"
+            res = await resp.json()
+            assert res["accessionId"] == study_access_id, "content mismatch"
+            assert res["alias"] == "GSE10966", "content mismatch"
+            assert res["descriptor"]["studyTitle"] == (
+                "Highly integrated epigenome maps in Arabidopsis - whole genome shotgun bisulfite sequencing"
+            ), "content mismatch"
+
+        """
+        import xml.etree.ElementTree as ET
+        LOG.debug("Sharing the correct accession ID created in this test instance")
+        mod_study = testfiles_root / "study" / "SRP000539_modified.xml"
+        tree = ET.parse(mod_study)
+        root = tree.getroot()
+        for elem in root.iter("STUDY"):
+            elem.set("accession", study_access_id)
+        tree.write(mod_study, encoding="utf-8")
+        """
```

Comment:
This is commented out because this part should cause a working test but doesn't. Requires some further solving

Reply:
it gives an error because i assume it tries to change `accessionID`
```
_format_data_to_update_and_add_to_db: Some items (e.g: accessionId, publishDate, dateCreated) cannot be changed.
```
When we do a `MODIFY` we should ignore `accessionID` field

I see `new_study_access_id = res[0]["accessionId"]` there in the test below :thinking: